### ISSUE TITLE
Please report this bug with ALL the details below, including both the 'Environment' and 'Exception' sections. Please report on GitHub: https://github.com/PowerShell/PSReadLine/issues/new?template=Bug_Report.md Thank you!

### ISSUE THREAD

**Issue by masud-rana-1** [PowerShell/PSReadLine#2969]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
### Environment
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.22000
BufferWidth: 120
BufferHeight: 30
```

### Screenshot

![image](https://user-images.githubusercontent.com/56246849/140903991-da849f18-9d6b-40a9-ad4f-e4383d0fa543.png)

### Environment data

```console
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.22000
BufferWidth: 120
BufferHeight: 30
```

### Steps to reproduce

new terminal update

### Expected behavior

![image](https://user-images.githubusercontent.com/56246849/140905144-0d941077-c6de-48a1-9bf4-21fc89dacfa3.png)

### Actual behavior

![image](https://user-images.githubusercontent.com/56246849/140905232-90b133f1-12bc-4d61-8a0a-6500476d2e1c.png)

**Comment by iSazonov** [comment]
Please update to latest Beta4 version.

**Comment by masud-rana-1** [comment]
no solve this issu

**Comment by masud-rana-1** [comment]
![image](https://user-images.githubusercontent.com/56246849/140912031-8fd2d116-1f6c-47bc-8722-2e9fa5900cd6.png)

**Comment by daxian-dbw** [comment]
This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4). Instructions for doing so:
1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.
